services/gateway/README.md

```diff
@@ -0,0 +1,94 @@
+# Lisk Service Gateway
+
+The Gateway service provides the API, which all users of Lisk Service can access and use. Its main purpose is to proxy API requests from users to other services provided by Lisk Service. This provides users with a central point of data access that never breaks existing application compatibility.
+
+> Note that this installation instruction is required only for the purpose of development activities. For a regular Lisk Service user the official [documentation](https://lisk.io/documentation/lisk-service/) is sufficient to run their own instance. The global readme file present in the root directory describes running all microservices at once.
+
+## Installation
+
+### Prerequisites
+
+Please refer to the readme file (`README.md`) in the project root directory.
+
+## Installation
+
+Clone the Lisk Service Repository:
+
+```bash
+git clone https://github.com/LiskHQ/lisk-service.git # clone repository
+cd lisk-service/components/gateway # move into core component folder
+npm install # install required Node.js modules
+```
+
+## Configuration
+
+To configure the different components, there are a number of environment variables, the user can define in order to customize the configurations.
+
+List of the most commonly used environment variables:
+
+- `SERVICE_BROKER`: URL of the microservice message broker (Redis)
+- `JSON_RPC_STRICT_MODE`: Makes JSON-RPC require a valid JSON-RPC 2.0 envelope
+- `ENABLE_HTTP_API`: Enables particular HTTP APIs listed by a comma
+- `ENABLE_WS_API`: Enables particular JSON-RPC APIs listed by a comma
+
+The variables listed above can be overridden globally by using global variables.
+
+```bash
+export ENABLE_HTTP_API="http-status,http-version2"
+```
+
+### Examples
+
+```bash
+# Run local instance with HTTP API only
+ENABLE_HTTP_API="http-status,http-version2" \
+ENABLE_WS_API="" \
+node app.js
+```
+
+```bash
+# Run local instance with the RPC API in a strict mode and HTTP
+ENABLE_HTTP_API="http-status,http-version2" \
+ENABLE_WS_API="blockchain,rpc-v2" \
+JSON_RPC_STRICT_MODE="true" \
+node app.js
+```
+
+## Management
+
+### Start
+
+```bash
+cd lisk-service/components/gateway # move into root folder of the core component
+npm start # start the component with running nodes locally
+```
+
+Use the `framework/bin/moleculer_client.js` and `framework/bin/moleculer_subscribe.js` clients to test particular service endpoints.
+
+If you want to run a production variant of the service use `Docker` or `PM2`. This will automatically recover the process when it fails.
```

Comment:
```suggestion
If you want to run a production variant of the service use `Docker` or `PM2`. In the event whereby the process fails, it will be automatically recovered.
```